pkg/display/tiledmap/ortho.go

```diff
@@ -16,7 +19,12 @@ type OrthoMap struct {
 }
 
 func NewOrthoMap(path string) *OrthoMap {
-	loadedMap, err := tiled.LoadFromFile(path)
+	bs, err := data.Asset(path)
+	if err != nil {
+		panic(err)
+	}
+
+	loadedMap, err := tiled.LoadFromReader(utils.FindPath("data/maps"), bytes.NewReader(bs))
```

Comment:
Need to find a way around resolving this to a real path and load subsequent files from bin data.

Reply:
I need to read more of the tiled source, but I think a new function `tiled.LoadFromReaderWithBindata` or something would let us get rid of the `baseDir` for loading subsequent files in the map.